opencti-platform/opencti-graphql/config/default.json

```
@@ -151,6 +151,9 @@
     "dissemination_list": {
       "to_email": "no-reply@filigran.io",
       "max_list_size": 500
+    },
+    "forgot_password": {
+      "otp_ttl": 600
```

```suggestion
      "otp_ttl_second": 600
```
Not mandatory but I personally like when I don't have to look for unit